lib/membrane_opus/encoder.ex

```diff
@@ -77,7 +77,8 @@ defmodule Membrane.Opus.Encoder do
 
   @impl true
   def handle_caps(:input, caps, _ctx, %{input_caps: caps} = state) do
-    {:ok, state}
+    output_caps = %Opus{channels: caps.channels}
+    {{:ok, caps: {:output, output_caps}}, state}
```

Comment:
Haven't we discussed that we should add `handle_prepared_to_playing` callback in which we will send caps if they were provided as element's option? 

Current approach should also work but I think it is less readable :thinking:

Reply:
Indeed, but I haven't changed that yet :D I have asked for your review even before our conversation